src/library/Uno.Toolkit.Material/MaterialToolkitResourcesV2.cs

```diff
@@ -91,42 +91,42 @@ private IEnumerable<(string ResourceKey, string SharedKey, bool IsDefaultStyle)>
 		{
 			var result = new List<(string ResourceKey, string SharedKey, bool IsDefaultStyle)>();
 			
-			Add("M3MaterialDividerStyle", isImplicit: true);
-			Add("M3MaterialNavigationBarStyle", isImplicit: true);
-			Add("M3MaterialModalNavigationBarStyle");
-			Add("M3MaterialMainCommandStyle", isImplicit: true);
-			Add("M3MaterialModalMainCommandStyle");
-			Add("M3MaterialTopTabBarStyle");
-			Add("M3MaterialColoredTopTabBarStyle");
-			Add("M3MaterialElevatedSuggestionChipStyle");
-			Add("M3MaterialSuggestionChipStyle");
-			Add("M3MaterialInputChipStyle");
-			Add("M3MaterialElevatedFilterChipStyle");
-			Add("M3MaterialFilterChipStyle");
-			Add("M3MaterialElevatedAssistChipStyle");
-			Add("M3MaterialAssistChipStyle");
-			Add("M3MaterialElevatedSuggestionChipGroupStyle");
-			Add("M3MaterialSuggestionChipGroupStyle");
-			Add("M3MaterialInputChipGroupStyle");
-			Add("M3MaterialElevatedFilterChipGroupStyle");
-			Add("M3MaterialFilterChipGroupStyle");
-			Add("M3MaterialElevatedAssistChipGroupStyle");
-			Add("M3MaterialAssistChipGroupStyle");
-			Add("M3MaterialBottomTabBarStyle");
-			Add("M3MaterialBottomTabBarItemStyle");
-			Add("M3MaterialBottomFabTabBarItemStyle");
-			Add("M3MaterialOutlinedCardStyle");
-			Add("M3MaterialFilledCardStyle");
-			Add("M3MaterialElevatedCardStyle");
-			Add("M3MaterialAvatarOutlinedCardStyle");
-			Add("M3MaterialAvatarFilledCardStyle");
-			Add("M3MaterialAvatarElevatedCardStyle");
-			Add("M3MaterialSmallMediaOutlinedCardStyle");
-			Add("M3MaterialSmallMediaFilledCardStyle");
-			Add("M3MaterialSmallMediaElevatedCardStyle");
-			Add("M3MaterialOutlinedCardContentControlStyle");
-			Add("M3MaterialFilledCardContentControlStyle");
-			Add("M3MaterialElevatedCardContentControlStyle");
+			Add("MaterialAssistChipGroupStyle");
```

Comment:
while it doesnt matter what order they are added here
i would prefer to group them by filename asc and in the same order as listed in the xaml
sort by alphabetically order make no-sense here...

Reply:
it is easier to check against changes that way; since it would be in document order